src/lib/reasoners/uf.ml

```diff
@@ -1188,8 +1188,7 @@ let compute_concrete_model_of_val cache =
        We should check if these symbols can be defined as solvable to
        remove this particular case here. *)
     if X.is_solvable_theory_symbol f || is_destructor f
-       || Sy.is_internal f || E.is_internal_name t || E.is_internal_skolem t
-       || E.equal t E.vrai || E.equal t E.faux
+       || Sy.is_internal f || E.equal t E.vrai || E.equal t E.faux
```

Comment:
`Sy.is_internal` is true for all names with a non-`User` namespace, so checking again for `Fresh` and `Skolem` namespaces with `E.is_internal_name` and `E.is_internal_skolem` is not necessary.